### DISCUSSION TITLE
Auto Update Date Elements via API

### DISCUSSION THREAD

**Discussion by sllewji** [xanthics/genshin_grind_planner#10] [Ideas]
I've settled on this planner after checking out a number of others, and it's been great.
One thing that I would really like to see is the relevant data elements such as inventory being automatically updated using one of the new API providers.

Would that be a possibility?

**Comment by xanthics** [comment]
Do you have a specific API in mind?  I can't find any Mihoyo supplied ones and am wary about implementing a 3rd party one.

**Comment by sllewji** [comment]
Hello Jeremy,

Thanks for getting back to me, so quickly.

I completely understand your reluctance to connect into a third party API provider, however they _must_ be connecting to some official source or other using stored credentials.

I note that the official Teyvet map site has a link to you own account, and the account page can provide details of your characters and some of your stats, and wonder whether this could be reversed engineered and mined for additional details on inventory and character details (I'm an MS SQL Server coder with very limited web experience).

It would make _such_ a difference to be the first planner site to be able to automatically update relevant data - even if you have to temporarily use a third party source.

Thanks for taking the time to respond!

Very Best Regards,
Ian Wells

Sent from my OnePlus phone

________________________________
